code/__DEFINES/~monkestation/cargo_import_companies.dm

```diff
@@ -0,0 +1,28 @@
+// Bitflags for what company a cargo order datum should belong to
+#define CARGO_COMPANY_NAKAMURA_MODSUITS (1<<0)
```

Comment:
you're adding guncargo w/ this btw

Reply:
yeah im still doing cleanup, im having trouble transferring files right now so its slow.